pkg/decision/composite_service_test.go

```diff
@@ -90,6 +97,37 @@ func (s *CompositeServiceFeatureTestSuite) TestDecisionListeners() {
 	s.Equal(numberOfCalls, 1)
 }
 
+func (s *CompositeServiceFeatureTestSuite) TestDecisionListenersNotificationContent() {
```

Comment:
Please also add a test case where we do not have variable info in the notification content

Reply:
addressed